components/TicketsSection.js

```diff
@@ -18,12 +18,16 @@ export default function TicketsSection () {
 function TicketsTable () {
   return (
     <div className='tickets-table'>
-      <TicketsRow title='1st round' date='18 May 2017' />
+      <TicketsRow className='soldout' title='1st round' date='18 May 2017' />
```

Comment:
I think this does not work, as the `TicketsRow` does not take a `className` props.

For a working solution, please check out [the changes in #43](https://github.com/reactbkk/2.0.0/pull/43/files).